src/liftwing_api/models/revertrisk.py

```diff
@@ -0,0 +1,44 @@
+import requests
+import json
+from liftwing_model import LiftwingModel
+
+class RevertRiskAPIModel(LiftwingModel):
+    def __init__(self, base_url="https://api.wikimedia.org/service/lw/inference/v1/models/{language}-reverted:predict"):
```

Comment:
It is a good idea to keep only as base url the first part of the string
`https://api.wikimedia.org/service/lw/inference/v1/models/`
and the latter part will be the one specific to the model and can be hardcoded for now like this:
```
self.model_name = "revertrisk-language-agnostic"
self.url = f"{base_url}{self.model_name}:predict"